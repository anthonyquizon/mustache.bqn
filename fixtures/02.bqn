data⇐{x⇐⟨{y⇐1⋄z⇐2},{y⇐3⋄z⇐4}⟩}
