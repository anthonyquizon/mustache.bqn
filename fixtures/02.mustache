<a>{{#x}}<b>{{y}} {{z}}</b>{{/x}}</a>
