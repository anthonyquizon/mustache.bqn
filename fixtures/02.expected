<a><b>1 2</b><b>3 4</b></a>
